pkg/util/gpu/paths.go

```diff
@@ -168,6 +170,11 @@ func paths(gpuFileList []string) ([]string, []string, error) {
 		return nil, nil, fmt.Errorf("could not lookup ldconfig: %v", err)
 	}
 	out, err := exec.Command(ldConfig, "-p").Output()
+	// #5002 If we failed and our ldconfig is not in standard POSIX location, try that
+	if err != nil && ldConfig != systemLdconfig {
```

Comment:
Not related to your changes but while we are here ... after looking at the code above, there is a strange path :
```
        ldConfig, err := exec.LookPath("ldconfig")
	if ee, ok := err.(*exec.Error); ok && ee.Err == exec.ErrNotFound {
		sylog.Debugf("Could not find ldconfig in PATH")
		ldConfig = "ldconfig"
	}
	if err != nil {
		return nil, nil, fmt.Errorf("could not lookup ldconfig: %v", err)
	}
```

which let think about a code artifact like :
```
        ldConfig, err := exec.LookPath("ldconfig")
	if ee, ok := err.(*exec.Error); ok && ee.Err == exec.ErrNotFound {
		sylog.Debugf("Could not find ldconfig in PATH")
		ldConfig = "ldconfig"
	} else if err != nil {
		return nil, nil, fmt.Errorf("could not lookup ldconfig: %v", err)
	}
``` 

Removing the useless statement `ldConfig = "ldconfig"` would avoid this confusion

Reply:
ahh - good catch, that is an odd flow there.